### ISSUE TITLE
How to disable hard wrapping in the editor?

### ISSUE THREAD

**Issue by earksiinni** [pazz/alot#1472]
Is there a way to disable hard wrapping in the editor? I'm not sure whether this is an issue with alot, though. I'm just using the default configuration and my `EDITOR` environment variable points to `nvim`.

**Comment by pazz** [comment]
I'm not sure I understand your question.
Does the default config together with your EDITOR env setting not work?


Quoting Ersin Akinci (2020-03-03 22:28:05)

**Comment by earksiinni** [comment]
When I edit a message, alot loads `nvim` because that's what my `EDITOR` environment variable is set to open. However, once inside `nvim`, my lines get hard wrapped even though I don't have anything in my vim config that hard wraps lines. My question was whether this has something to do with alot and, if so, how I can change it.

I looked into it a bit further and Neovim was detecting the file type of my messages as `mail` based on the `.eml` filename extension. I'm guessing that something inside Neovim's default behavior causes any mail-type files to automatically run `set textwidth=72`, which was the specific setting that was causing the hard wrapping. I was able to fix this issue by adding `autocmd FileType mail setlocal textwidth=0` to my vimrc.

At any rate, this wasn't an issue with alot.
